src/frontend/@types/mapeo-schema.d.ts

```diff
@@ -0,0 +1,147 @@
+// Type definitions for mapeo-schema 3.0.0
+// Project: https://github.com/digidem/mapeo-schema/
+// Definitions by: Andrew Chou <https://github.com/achou11>
+
+declare module "mapeo-schema" {
```

Comment:
This file be removed when updating to the next major version of `mapeo-schema` (see https://github.com/digidem/mapeo-schema/pull/13)